tests/Hangfire.PostgreSql.Tests/Utils/ConnectionUtils.cs

```diff
@@ -10,9 +10,10 @@ internal static class ConnectionUtils
         private const string ConnectionStringVariableName = "Hangfire_PostgreSql_ConnectionString";
 
         private const string DefaultConnectionString =
-            @"Server=localhost;Port=5432;Database=hangfire_tests;User Id=postgres;Password=password;Search Path=hangfire";
+            @"Server=docker.cocowalla.ga;Port=5432;Database=hangfire_tests;User Id=postgres;Password=password;Search Path=hangfire";
```

Comment:
Leave `localhost` as default value. You can pass custom connection string via Environment variable.